packages/paste-libraries/data-visualization/stories/index.stories.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import type {Story, Meta} from '@storybook/react';
+import {Stack} from '@twilio-paste/stack';
+/* eslint-disable import/no-extraneous-dependencies */
+import Highcharts from 'highcharts';
+import HighchartsReact from 'highcharts-react-official';
+/* eslint-enable */
+
+import {usePasteHighchartsTheme} from '../src';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Libraries/data-visualization',
+} as Meta;
+
+const lineChartOptions: Highcharts.Options = {
```

Comment:
Yeah, I get it, but I do still think it's worthwile expanding coverage.

For the hook, it might seem like a safe assumption that the chart implementation would cover it, but that assumes you're not doing something unexpected with it before it's fed to the chart. we'd just be ensuring the contract between the input and ouput of the hook stays the same, which might become super important later if we tweak anything in the guts of the function. They are peer dependencies to each other after all, so they should be treated with isolation.

For the charts, I think even though they are tested, it is probably worth while to ensure the changes we make to chart styles have the expected results on chart types that we aren't using in stories. Just to be extra cautious.